sql_user_check_looker/README.md

```diff
@@ -0,0 +1,11 @@
+# Check permissions on Looker instance
+
+
+### Run the script
+
+Running the script will print models and explores count which user has access to. Looker instance url, client_id and client_secret
+are inputs to this script. URL looks something like this https://23151ff5-b9c5-4327-9095-2cdaa11c1e43.looker.app
```

Comment:
Remove the url and give a template like <instanceId>.looked.app

Reply:
done.